packages/node/src/utils/ensureIsWrapped.ts

```diff
@@ -24,5 +24,11 @@ export function ensureIsWrapped(
         );
       }
     });
+
+    getGlobalScope().setContext('Instrumentation', {
```

Comment:
```suggestion
    getGlobalScope().setContext('instrumentation', {
```

If we are adding new context, let's please make sure this is documented in https://develop.sentry.dev/sdk/event-payloads/contexts/

I also fear that `instrumentation` is a pretty big namespace to take, and could collide with custom contexts relatively easily. Could we think about a name that potentially has less collisions?

We can discuss naming, and evaluate what exact fields should be here in the PR to develop https://develop.sentry.dev/sdk/event-payloads/contexts/. We might want to re-evaluate `isCjs` for example because it is JS specific, and contexts should be agnostic to language as much as possible.

Reply:
@AbhiPrasad

What about `missing_instrumentation`?

If we exclude the `isCjs`, it might make sense to replace the new context altogether with just a tag?

Reply:
Or have a generic `extra` object? Not sure if that's compatible tho, just thinking out loud.

Reply:
I like `missing_instrumentation`. `isCjs` does seems important, but we can get around this by namespacing the context keys with `javascript.X` to indicate language specific instrumentation errors.

contexts also tend to follow snake case as they are formatted for the backend.

```
interface MissingInstrumentationContext {
  // Note: This seems redundant because the existence of `missing_instrumentation` context should
  // indicate `is_missing: true`
  is_missing: boolean;
  // package name
  // this should probably be required
  package: string;
  ['javascript.is_cjs']?: boolean;
}
```

The SDK should avoid setting tags as much as possible because tags are meant for user-set values. In fact tags we "automatically" add like `browser` and `os` are actually converted from contexts to tags by the sentry backend during event ingestion. Tags are indexed and searchable in our backend, so setting new one's automatically puts a burden on the infrastructure that we'd much rather be able to control dynamically (hence the mechanism to "promote" certain contexts to tags like we do for `browser`).

`extra` also makes sense, but it is soft-deprecated, and the product/relay does not have code (like it does for contexts) to handle it. 

Hence I think picking contexts is the right vehicle for this because we can easily adapt relay/frontend/backend in the product-side to read contexts and use them, and it makes sense for the product to use these values for better onboarding/troubleshooting.

Reply:
Got it, thanks for the clarification on tags!